web/src/client/software.js

```diff
@@ -348,6 +246,73 @@ class ProductBaseClient {
       }
     });
   }
+
+  /**
+   * Returns the registration of the selected product.
+   *
+   * @return {Promise<Registration>}
+   */
+  async getRegistration() {
+    const response = await this.client.get("/software/registration");
+    if (!response.ok) {
+      console.log("Failed to get registration config: ", response);
+      return { requirement: "unknown", code: null, email: null };
+    }
+    const config = await response.json();
+
+    const requirement = config.requirement;
```

Comment:
You can use destructuring to get the date like this:

```
    const { requirement, email, code } = config;
```

Reply:
you are right, as conversion for requirement is no longer there, it is not needed